os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -77,12 +77,27 @@ config AMEBASMART_SPI0
 config AMEBASMART_SPI1
 	bool "SPI1"
 	default n
-
 config AMEBASMART_SPI_EXCHANGE
 	bool "Enable AMEBASMART SPI EXCHANGE"
 	default y
-endif
 
+config SPI_MASTER_WRITE_DISABLE_READ
```

Comment:
1. All others have `AMEBASMART_` prefix, but this is not. How about adding that here as well?
2. If I enable this, but disable AMEBASMART_SPI0_MASTER_WRITE_DISABLE_READ and AMEBASMART_SPI1_MASTER_WRITE_DISABLE_READ, what happen?
Do we need `SPI_MASTER_WRITE_DISABLE_READ` separately?

Reply:
Modification done